source/codegen/metadata/nidcpower/config.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# This file is generated from NI-DCPower API metadata version 20.7.0d8
+config = {
+    'api_version': '20.7.0d8',
+    'c_header': 'nidcpower.h',
+    'c_function_prefix': 'niDCPower_',
+    'service_class_prefix': 'NiDCPower',
+    'java_package': 'com.ni.grpc.dcpower',
+    'csharp_namespace': 'NationalInstruments.Grpc.DCPower',
+    'namespace_component': 'nidcpower',
+    'close_function': 'close',
```

Comment:
```suggestion
    'close_function': 'Close',
```
Match the capitalization used in `functions.py` for the close method so it can find it through the interface. I believe that's why the builds are failing currently.

Reply:
@reckenro, Thanks for putting this to notice, I changed them and now the build is also successful.